daemon/daemon.go

```diff
@@ -444,6 +466,17 @@ func (srv *shutdownServer) finishShutdown() error {
 	return fmt.Errorf("cannot gracefully finish, still active connections on %v after %v", srv.l.Addr(), shutdownTimeout)
 }
 
+func (d *Daemon) initStandbyHandling() {
+	d.standbyHelper = standby.New(d.overlord.State())
+	d.standbyHelper.AddOpinion(d.snapdServe)
+	d.standbyHelper.AddOpinion(d.snapServe)
+	d.standbyHelper.AddOpinion(d.overlord)
+	d.standbyHelper.AddOpinion(d.overlord.SnapManager())
+	d.standbyHelper.AddOpinion(d.overlord.DeviceManager())
+	// loop runs in its own go-routine
+	d.standbyHelper.Loop()
```

Comment:
The comment is only necessary because the method tastes like it's going to run the loop. If it's instead named as _Start_, we don't need the comment as it's clear that something else will keep going. It also makes it clear that something else will need to _Stop_ it, which I can't find here?